scripts/deploy.sh

```diff
@@ -40,6 +41,10 @@ ansible-galaxy install --role-file=/opt/rpc-openstack/ansible-role-requirements.
 # Enable playbook callbacks from OSA to display playbook statistics
 grep -q callback_plugins playbooks/ansible.cfg || sed -i '/\[defaults\]/a callback_plugins = plugins/callbacks' playbooks/ansible.cfg
 
+if [[ "$DEPLOY_MAGNUM" == "yes" ]]; then
```

Comment:
I would prefer using the ansible-role-requirements method :) See comments below.